examples/end-to-end-examples/sec_10k_qa/README.md

```diff
@@ -0,0 +1,190 @@
+
+Note: this step will take a number of hours (~19hrs on 8xA100_80GB). Either use the `small_full` data from previous steps, or change `max_duration` in the YAML to something smaller if you would like a shorter training run.
+
+**Fields to replace with your values:** `REPLACE_WITH_YOUR_CLUSTER`, `CLOUD`, `BUCKET_NAME`
+
+**Inputs:** the `sec-10ks-large-mds` folder from step 2
+
+**Command:** `mcli run -f mcli-yamls/03_finetune_on_10ks.yaml`
+
+**Outputs:** the checkpoints from your training, saved to the `save_folder` specified in the yaml
+
+
+## 4) Instruct Finetune MPT-7b
+
+Now that we have trained our model on in-domain financial text, we will train it to better follow instructions. For this step, we will follow the exact same process that we used to create [`mpt-7b-instruct`](https://huggingface.co/mosaicml/mpt-7b-instruct). Namely, we will finetune the model on instruction formatted data following the form of the [`databricks-dolly-15k` dataset](https://huggingface.co/datasets/databricks/databricks-dolly-15k), shown below. See [`mosaicml/dolly_hhrlhf`](https://huggingface.co/datasets/mosaicml/dolly_hhrlhf) for more details on the specific dataset we are using.
+
+```python
+INSTRUCTION_KEY = "### Instruction:"
+RESPONSE_KEY = "### Response:"
+INTRO_BLURB = "Below is an instruction that describes a task. Write a response that appropriately completes the request."
+PROMPT_FOR_GENERATION_FORMAT = """{intro}
+{instruction_key}
+{instruction}
+{response_key}
+""".format(
+    intro=INTRO_BLURB,
+    instruction_key=INSTRUCTION_KEY,
+    instruction="{instruction}",
+    response_key=RESPONSE_KEY,
+)
+
+example = "James decides to run 3 sprints 3 times a week. He runs 60 meters each sprint. How many total meters does he run a week? Explain before answering."
+fmt_ex = PROMPT_FOR_GENERATION_FORMAT.format(instruction=example)
+```
+
+For this second finetuning step, we will use the same training script as before, but with a different training yaml. The [training yaml](./mcli-yamls/04_instruction_finetune_on_dolly_hh.yaml) will load the same model class and architecture from the HuggingFace Hub, but will set `pretrained: false` and instead load the weights from the Composer checkpoint from the previous step using `load_path`. See this [finetuning yaml](https://github.com/mosaicml/llm-foundry/blob/main/scripts/train/yamls/finetune/7b_dolly_sft.yaml) for an example of how to finetune directly with Composer without using HuggingFace Hub (note the difference in the `model` section).
+
+**Fields to replace with your values:** `REPLACE_WITH_YOUR_CLUSTER`, `CLOUD`, `BUCKET_NAME`, `PREVIOUS_RUN_NAME`
+
+**Inputs:** the final checkpoint from step 3
+
+**Command:** `mcli run -f yamls/mcli/04_instruction_finetune_on_dolly_hh.yaml`
```

Comment:
Make this a code block so people can hit copy button
```suggestion
**Command:** 
```bash
mcli run -f yamls/mcli/04_instruction_finetune_on_dolly_hh.yaml
```
```